From 95ea1ec630e016d5cb190266c518d8716c80e530 Mon Sep 17 00:00:00 2001
From: Brandon Boese <brandonboese@protonmail.com>
Date: Sun, 15 Oct 2023 17:49:13 -0400
Subject: [PATCH] remove daemon directory

Temporary patch to allow compiling. Will be fixed when work on ofono begins.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ffd6a93..6267ce4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -97,7 +97,7 @@ set(HISTORY_VERSION_PATCH 0)
 set(PACKAGE_VERSION ${HISTORY_VERSION_MAJOR}.${HISTORY_VERSION_MINOR}.${HISTORY_VERSION_PATCH})
 
 add_subdirectory(src)
-add_subdirectory(daemon)
+# add_subdirectory(daemon)
 add_subdirectory(plugins)
 add_subdirectory(tools)
 add_subdirectory(Lomiri)
-- 
2.42.0

